sdcm/nemesis.py

```diff
@@ -250,14 +242,27 @@ def repair_nodetool_rebuild(self):
         self._run_nodetool(rebuild_cmd, self.target_node)
 
 
-def log_time_elapsed(method):
+def log_time_elapsed_and_status(method):
     """
     Log time elapsed for method to run
 
     :param method: Remote method to wrap.
     :return: Wrapped method.
     """
+    def print_nodetool_status(self):
+        for node in self.cluster.nodes:
```

Comment:
Yes, but asserting will Fail the entire test.
on one hand, I'm in favor so we can login and start to investigate.
OTOH, we don't have any assert at the moment, and maybe it should be part of a bigger task that will be tested separately.